pkgs/development/python-modules/python-mapnik/default.nix

```diff
@@ -17,37 +19,42 @@
 , mapnik
 , proj
 , zlib
+, libxml2
+, sqlite
 }:
 
 buildPythonPackage rec {
   pname = "python-mapnik";
-  version = "unstable-2020-02-24";
+  version = "unstable-2020-09-08";
 
   src = fetchFromGitHub {
     owner = "mapnik";
     repo = "python-mapnik";
-    rev = "7da019cf9eb12af8f8aa88b7d75789dfcd1e901b";
-    sha256 = "0snn7q7w1ab90311q8wgd1z64kw1svm5w831q0xd6glqhah86qc8";
+    rev = "a2c2a86eec954b42d7f00093da03807d0834b1b4";
+    sha256 = "sha256-GwDdrutJOHtW7pIWiUAiu1xucmRvp7YFYB3YSCrDsrY=";
+    fetchSubmodules = true;
```

Comment:
Maybe add a comment that this is only used for the test data.

Reply:
Done :+1:

Reply:
The hash doesn't match.  You surely need to update it when switching `fetchSubmodules`.